resources/guidelines/testing/store/quality-guidelines-plugins/index.md

```diff
@@ -113,6 +113,7 @@ However, you may employ `<span class="h2">`, for instance.
 However, it is difficult to read and understand the compiled code.
 To ensure that the code remains accessible to all developers, the uncompiled version of the JavaScript code must be placed in a separate folder.
 This allows other developers to review and understand the code in its original, readable form.
+Shopware reserves the right to publish extensions with minified code after individual consideration and consultation with the developer. For this, the developer must ensure that Shopware has access to the current unminified code of the extension at all times.
```

Comment:
@sdenter please add "unminified" to the spellcheck. how to here -> https://github.com/shopware/docs?tab=readme-ov-file#spellcheck